package.json

```diff
@@ -71,6 +71,9 @@
   "ember-addon": {
     "configPath": "tests/dummy/config",
     "demoURL": "https://miguelcobain.github.io/ember-paper",
-    "before": "ember-cli-sri"
+    "before": "ember-cli-sri",
+    "versionCompatibility": {
+      "ember": ">=2.3.0"
```

Comment:
Could you please point me to the documentation for this option within `ember-addon`. Google failed me. Or maybe I failed google. ;)